fixtures/directives.math.md

```diff
@@ -0,0 +1,22 @@
+Math directive:
+.
+```{math}
+:label: my_label
+w_{t+1} = (1 + r_{t+1}) s(w_t) + y_{t+1}
+```
+.
+<div class="math numbered" id="eq-my_label" number="1">
+w_{t+1} = (1 + r_{t+1}) s(w_t) + y_{t+1}
+</div>
```

Comment:
Right now the math doesn't add `\[` around it. I am not sure we actually want this? The alternative is to use something else to notify mathjax.